react/components/ProductPrice/README.md

```diff
@@ -14,28 +14,86 @@ You can use it in your code like a React component with the jsx tag: `<ProductPr
+  showLabels={showLabels}
+  showListPrice={showListPrice}
+  showInstallments={showInstallments}
   sellingPrice={commertialOffer.Price}
+  listPrice={commertialOffer.ListPrice}
   installments={commertialOffer.Installments}
-  showListPrice={showListPrice}
+/>
+```
+
+### Passing classes to the elements of the component
+The `classes` and the `installmentsClasses` props have the following structure
+```js
+{
+  classes: PropTypes.shape({
+    root: PropTypes.string,
+    rootLoader: PropTypes.string,
+    listPrice: PropTypes.shape({
+      container: PropTypes.string,
+      label: PropTypes.string,
+      value: PropTypes.string
+    }),
+    sellingPrice: PropTypes.shape({
+      container: PropTypes.string,
+      label: PropTypes.string,
+      value: PropTypes.string
+    }),
+    savings: PropTypes.shape({
+      container: PropTypes.string,
+      value: PropTypes.string
+    })
+  }),
+  installmentsClasses: PropTypes.shape({
+      root: PropTypes.string,
+      installmentValue: PropTypes.string,
+      interestRate: PropTypes.string
+  })
+}
+```
+The attributes of `classes` and `installmentsClasses` objects represent an element of the component. To understand better see the following example of how to pass classes to elements
+```jsx
+
+const classes = {
+  root: 'vtex-price some-css-class',
+  rootLoader: 'vtex-price vtex-price-loader some-other-css-class',
+  sellingPrice: {
+    container: 'some-css-class-for-selling-container'
+  }
+}
+
+const installmentsClasses = {
+  root: 'css-class-for-root-of-installments'
+}
+
+<ProductPrice
+  classes={classes}
   showLabels={showLabels}
+  showListPrice={showListPrice}
   showInstallments={showInstallments}
+  sellingPrice={commertialOffer.Price}
+  listPrice={commertialOffer.ListPrice}
+  installmentsClasses={installmentsClasses}
+  installments={commertialOffer.Installments}
 />
 ```
 
-| Prop name | Type | Description |
-| --- | --- | --- |
-| `sellingPrice` | `Number` | Product selling price |
-| `listPrice` | `Number` | Product list price |
-| `showListPrice` | `Boolean!` | Set visibility of list price |
-| `showLabels` | `Boolean!` | Set visibility of labels |
-| `showInstallments` | `Boolean!` | Set visibility of installments |
-| `showSavings` | `Boolean` | Set visibility of savings |
-| `installments` | `Array` | Available installments |
-| `installments.Value` | `Number!` | Installment price |
-| `installments.InterestRate` | `Number!` | Installment interest rate (zero if interest-free) |
-| `installments.TotalValuePlusInterestRate` | `Number` | Calculated total value |
-| `installments.NumberOfInstallments` | `Number!` | Number of installments |
-| `installments.Name` | `String` | Installment offer name |
-
-See an example at [Product Summary](https://github.com/vtex-apps/product-summary/blob/master/react/ProductSummary.js#L84) app
+| Prop name                                 | Type       | Description                                                |
```

Comment:
Remove the spaces from this table, as it makes really bad to maintain these docs